src/DOM/HTML/History.js

```diff
@@ -0,0 +1,47 @@
+// module DOM.HTML.History
```

Comment:
We can drop these comments now, the compiler switched to using paths to figure out associated FFI files a few versions ago.